database/scripts/import/insert_srid_54009.sql

```diff
@@ -1,14 +0,0 @@
-/* Required for GHS_SMOD raster */
```

Comment:
not sure if we might still need this in postgres? How do we query the ghspop or ghssmod rasters with rasterstats? Do we need to tranform the region geometry into 54009 first?

Reply:
We do not need it at the moment. 
The OQT raster client (`rasterstats`) will transfer geometries using `pyproj` if necessary (no database involved in this step).